cybsi/api/threat_landscapes/api.py

```diff
@@ -0,0 +1,590 @@
+import uuid
+from typing import Any, Dict, Optional
+
+from cybsi.api import RefView, Tag
+from cybsi.api.custom_list import CustomListCommonView
+from cybsi.api.internal import BaseAPI, BaseAsyncAPI, JsonObjectForm
+from cybsi.api.pagination import AsyncPage, Cursor, Page
+from cybsi.api.view import _TaggedRefView
+
+_PATH = "threat-landscapes"
+
+
+class ThreatLandscapesAPI(BaseAPI):
+    """
+    API to operate Threat Landscapes.
+
+    .. versionadded:: 2.14.0
+    """
+
+    def filter(
+        self,
+        *,
+        cursor: Optional[Cursor] = None,
+        limit: Optional[int] = None,
+    ) -> Page["ThreatLandscapeCommonView"]:
+        """
+        Get a list of all threat landscapes.
```

Comment:
Минор, как-то странно смотриться, может без `all`
```suggestion
        Get a list of threat landscapes.
```